scripts/module/teleporter.nut

```diff
@@ -36,6 +36,8 @@ function onPlayerCommand( player, cmd, text ) {
     } else {
       MessagePlayer("Usage: /" + cmd + " <string>", player);
     }
+  } else if (cmd == "help") {
+    MessagePlayer("[#333380]/t", player);
```

Comment:
For these short forms we should probably use something like `/t[eleport]` and accept both commands in the command handler. Can be done in a separate PR.